Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -91,31 +92,34 @@ static int GetTypeSize(MemoryViewWidget::Type type)
 
 static int GetCharacterCount(MemoryViewWidget::Type type)
```

Comment:
If you plan to include the space, what about `GetCellSize` as a function name? Can't this function be marked as `constexpr`. I also highly suggest that for each case, you write an example.

For instance:
```c++
case MemoryViewWidget::Type::Unsigned32: // UINT_MAX = 4294967295
  return 11;
case MemoryViewWidget::Type::Float32: // FLT_MAX = 3.40282e+38
case MemoryViewWidget::Type::Signed32: // INT_MIN = -2147483648
  return 12;
```

Reply:
Yeah I don't think it will be used for anything else. I can add examples too.

Reply:
Actually the cell size needs to be calculated from the font, so I think CharacterCount is most accurate.